package.json

```diff
@@ -112,4 +112,4 @@
       "npm run test_stylelint"
```

Comment:
Here we need also to run `prettier --write` as for `*.ts` files, so staged files will be autoformatted

Reply:
Yes, it for the cases when someone doesn't use VS Code workspace or using another editor

Reply:
Sounds good. Please review again